lib/lazy_high_charts/layout_helper.rb

```diff
@@ -106,11 +106,12 @@ def encapsulate_js(core_js)
         #{js_end}
         EOJS
       # Turbolinks >= 5
-      elsif defined?(Turbolinks) && request_turbolinks_5_tureferrer?
+      elsif defined?(Turbolinks) && is_turbolinks_5?
         js_output =<<-EOJS
         #{js_start}
-          document.addEventListener("turbolinks:load", function() {
+          document.addEventListener("turbolinks:load", function(e) {
             #{core_js}
+          	e.target.removeEventListener(e.type, arguments.callee);
```

Comment:
interesting, can you explain to us what this code does. where is the `arguments` variable is defined ?

Reply:
Any chance you can merge this fix without the rest of the PR?  This fix is really important, but on the other hand other commits from this PR break my code (e.g. verifying how to encapsulate JS only based on the Turbolinks v5 gem being loaded into the application)